src/bmp/message.rs

```diff
@@ -857,24 +857,80 @@ impl<Octs: Octets> PeerUpNotification<Octs> {
         (sent, rcvd) 
     }
 
-    /// Create a [`SessionConfig`] describing the parameters for the BGP
+    /// Create a [`SessionConfig`] describing the parameters for the BMP
     /// session between the monitored router and the remote peer.
```

Comment:
SessionConfig really is used to parse BGP PDUs, even though some of its settings might come from flags in a BMP header and the BGP PDUs are encapsulated in BMP. So perhaps:


```suggestion
    /// Create a [`SessionConfig`] to parse encapsulated BGP data based on the PerPeerHeader.
    ///
```